Makefile

```diff
@@ -37,31 +38,31 @@ compile-pip-dependencies: ## Uses pip-compile to update requirements.txt
 # It is critical that we run pip-compile via the same Python version
 # that we're generating requirements for, otherwise the versions may
 # be resolved differently.
-	docker run -v "$(DIR)/securethenews:/code" -w /code -it python:3.6-slim \
+	docker run -v "$(DIR):/code" -w /code/securethenews -it $(PY_IMAGE) \
 		bash -c 'apt-get update && apt-get install gcc -y && \
-    pip install --require-hashes -r /code/dev-requirements.txt && \
-		pip-compile --generate-hashes --no-header --output-file /code/requirements.txt /code/requirements.in && \
-		pip-compile --generate-hashes --no-header --allow-unsafe --output-file /code/dev-requirements.txt /code/dev-requirements.in'
+		pip install pip-tools && \
+		pip-compile --generate-hashes --no-header --output-file requirements.txt requirements.in && \
+		pip-compile --generate-hashes --no-header --allow-unsafe --output-file dev-requirements.txt dev-requirements.in'
 
 .PHONY: pip-update
-upgrade-pip: ## Uses pip-compile to update requirements.txt for upgrading a specific package
+pip-update: ## Uses pip-compile to update requirements.txt for upgrading a specific package
 # It is critical that we run pip-compile via the same Python version
 # that we're generating requirements for, otherwise the versions may
 # be resolved differently.
-	docker run -v "$(DIR):/code" -w /code -it python:3.6-slim \
+	docker run -v "$(DIR):/code" -w /code/securethenews -it $(PY_IMAGE) \
 		bash -c 'apt-get update && apt-get install gcc -y && \
-    pip install --require-hashes -r dev-requirements.txt && \
+		pip install pip-tools && \
 		pip-compile --generate-hashes --no-header --upgrade-package $(PACKAGE) --output-file requirements.txt requirements.in && \
 		pip-compile --generate-hashes --no-header --allow-unsafe --upgrade-package $(PACKAGE) --output-file dev-requirements.txt dev-requirements.in'
 
 .PHONY: pip-dev-update
-update-pip-dev: ## Uses pip-compile to update dev-requirements.txt for upgrading a specific package
+pip-dev-update: ## Uses pip-compile to update dev-requirements.txt for upgrading a specific package
```

Comment:
This is also great! It fixes something that was totally my fault and continued to annoy me every time I noticed it. Thank you!